docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -12,56 +12,61 @@ description: Asset jobs are the main unit for materializing and monitoring Softw
 
----
+```python file=/concepts/assets/non_argument_deps.py startafter=start_marker endbefore=end_marker
+from dagster import asset
 
-## Creating asset jobs
 
-Asset jobs materialize a fixed set of assets each time they run. Additionally, multiple jobs can target overlapping sets of assets:
+@asset
+def sugary_cereals() -> None:
+    execute_query(
+        "CREATE TABLE sugary_cereals AS SELECT * FROM cereals WHERE sugar_grams > 10"
+    )
 
-```python file=/concepts/assets/build_job.py startafter=start_job_only_marker endbefore=end_job_only_marker
-from dagster import Definitions, asset, define_asset_job
 
+@asset(deps=[sugary_cereals])
+def shopping_list() -> None:
+    execute_query("CREATE TABLE shopping_list AS SELECT * FROM sugary_cereals")
+```
 
-@asset
-def asset1():
-    return [1, 2, 3]
+To create an asset job, use <PyObject object="define_asset_job" />. An asset-based job is based on the [assets](/concepts/assets/software-defined-assets) the job targets and their dependencies.
 
+You can target one or multiple assets, or create multiple jobs that target overlapping sets of assets. In the following example, we have two jobs:
 
-@asset
-def asset2(asset1):
-    return asset1 + [4]
+- `all_assets_job` targets all assets
+- `sugary_cereals_job` targets only the `sugary_cereals` asset
+
+```python file=/concepts/assets/build_job.py startafter=start_marker endbefore=end_marker
+from dagster import Definitions, define_asset_job
 
 
 all_assets_job = define_asset_job(name="all_assets_job")
-asset1_job = define_asset_job(name="asset1_job", selection="asset1")
+sugary_cereals_job = define_asset_job(
+    name="sugary_cereals_job", selection="sugary_cereals"
+)
 
 defs = Definitions(
-    assets=[asset1, asset2],
-    jobs=[all_assets_job, asset1_job],
+    assets=[sugary_cereals, shopping_list],
+    jobs=[all_assets_job, sugary_cereals_job],
 )
 ```
 
-The topology of an asset-based job is based on the [assets](/concepts/assets/software-defined-assets) and their dependencies.
-
 ---
 
 ## Making asset jobs available to Dagster tools
 
-You make asset jobs available to the UI, GraphQL, and the command line by including them in a <PyObject object="Definitions"/> object at the top level of a Python module or file. The tool loads that module as a code location. If you include schedules or sensors, the [code location](/concepts/code-locations) will automatically include jobs that those schedules or sensors target.
+Including the jobs in a <PyObject object="Definitions"/> object located at the top level of a Python module or file makes asset jobs available to the UI, GraphQL, and the command line. The Dagster tool loads that module as a code location. If you include schedules or sensors, the [code location](/concepts/code-locations) will automatically include jobs that those schedules or sensors target.
 
 ```python file=/concepts/assets/jobs_to_definitions.py
```

Comment:
I'm not sure how exhaustive you want to be about converting examples away from I/O managers, but if you want to get everything, then this one should be updated to not return anything

Reply:
Updated the example - I think I did this correctly 😅